stdlib/_typeshed/importlib.pyi

```diff
@@ -0,0 +1,18 @@
+# Implicit protocols used in importlib.
+# We intentionally omit deprecated and optional methods.
+
+from collections.abc import Sequence
+from importlib.machinery import ModuleSpec
+from types import ModuleType
+from typing import Protocol
+
+__all__ = ["LoaderProtocol", "MetaPathFinderProtocol", "PathEntryFinderProtocol"]
+
+class LoaderProtocol(Protocol):
```

Comment:
You should be able to replace `types._LoaderProtocol` with this as well.

Reply:
Replacing that would create an import loop between `types.pyi` and `_typeshed/importlib.pyi`. Is that OK to do for type stubs?

Reply:
Loops are not a problem in stubs.

Reply:
Thank you for the clarification, I introduced this change in 7075e5e.